### PR TITLE
MedKit

### PR DESCRIPTION
**Pull request by KevinCabrera25**
Functional MedKits have been implemented in this version.

Whenever the Player enters in the OnTriggerEvent a CompareTag condition is used to analyse if the MedKit has been found, and if so the prop is destroyed after adding a 30 value to the Player's HP.

### CODE REVIEW

File: Assets/_Main/Scripts/PlayerHP.cs [hunk 1]
```diff
@@ -29,4 +34,25 @@ public void GameOver()
         // Invokes the Method Screen from the GameOverScreen Script
         GameOverScreen.Screen();
     }
+
+    private void OnTriggerEnter(Collider other)
+    {
+        if (other.CompareTag("MedKit"))
+        {
+            // Invokes the Coroutine Grab
+            StartCoroutine(Grab(other.GetComponent<MeshRenderer>()));
```
Comment (Horothenic) [inline Assets/_Main/Scripts/PlayerHP.cs:43]: You are painting the medkit?

File: Assets/_Main/Scripts/PlayerHP.cs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 using System.Collections;
 using System.Collections.Generic;
```
Comment (Horothenic) [inline Assets/_Main/Scripts/PlayerHP.cs:2]: Remove unused libraries

File: Assets/_Main/Scripts/PlayerHP.cs [hunk 3]
```diff
@@ -9,12 +10,16 @@ public class PlayerHP : MonoBehaviour
     [SerializeField] private float _playerHP;
     // Reference to assign the GAME OVER SCREEN
     [SerializeField] private GameOverScreen GameOverScreen;
+    // Reference to assign the HP UI
+    [SerializeField] private TextMeshProUGUI _playerHPUI;
```
Comment (Horothenic) [inline Assets/_Main/Scripts/PlayerHP.cs:14]: Initialize

**Review by Horothenic — CHANGES_REQUESTED** [review]

File: Assets/_Main/Scripts/PlayerHP.cs (hunk 2 above)
Comment (KevinCabrera25) [reply Assets/_Main/Scripts/PlayerHP.cs]: Done

File: Assets/_Main/Scripts/PlayerHP.cs (hunk 3 above)
Comment (KevinCabrera25) [reply Assets/_Main/Scripts/PlayerHP.cs]: Done

**Review by Horothenic — APPROVED** [review]

**Pull request merged**